packages/docs/docs/provider/set-up.md

```diff
@@ -0,0 +1,71 @@
+# Set Up
+
+This section covers the initial setup process for your Medplum system, including how to add practitioners and import existing data through the Medplum App.
+
+<br />
+
+--- 
+
+## **Adding Practitioners (via Medplum App)**
+
+Setting up practitioners is one of the first steps in configuring your Medplum system and will need to be completed using the Medplum App (app.medplum.com). Practitioners represent healthcare providers, clinicians, and other staff members who will be using the system.
+
+### **Prerequisites**
+
+* Administrative access to the Medplum App  
+* Practitioner information including names, credentials, and contact details  
+* Valid email addresses for each practitioner (required for system access)
+
+### **How to Add Practitioners**
+
+1. **Access the Practitioners Section**  
+   * Log into the Medplum App with your administrator credentials  
+   * Select "Practitioner" from the left navigation menu  
+2. **Create a New Practitioner**  
+   * On the Practitioners page, click the "New…" action button at the top of the page  
+   * Fill in the required practitioner information  
+3. **Verify and Save**  
+   * Review all entered information and then click "Create” or "Create Practitioner" to complete the setup  
+   * The system will generate a unique practitioner ID automatically  
+4. **Share Credentials**  
```

Comment:
An invite can/will be sent to the practitioner asking them to log in.  The invite will be sent to app.medplum.com - this is perhaps a rough edge.